sdk/test/trace/sampler_benchmark.cc

```diff
@@ -0,0 +1,132 @@
+#include "opentelemetry/sdk/trace/samplers/always_off.h"
```

Comment:
It's not uncommon (within Google) to include the results of running the benchmark in a comment at the top of the file.

Reply:
Good to know, thanks. For this situation would you recommend sticking with OTel's existing convention of not having the results in the file, or should I talk to Johannes and get his thoughts on potentially adding this to all benchmark files?

Reply:
We can defer to Johannes. It's probably more important to be consistent with the OpenTelemetry code.

Reply:
@pyohannes Thoughts? I've left the results of the benchmark in the comments of this PR for reference

Reply:
We store benchmark results as artifacts for the [Benchmark CI job](https://github.com/open-telemetry/opentelemetry-cpp/pull/175/checks?check_run_id=889516416). It's an interesting idea to have results at the top of the file, but we haven't adopted this convention for this project, so no need to add it.